numpy/_core/src/multiarray/nditer_constr.c

```diff
@@ -492,6 +491,11 @@ NpyIter_AdvancedNew(int nop, PyArrayObject **op_in, npy_uint32 flags,
             }
         }
     }
+    else if (itflags&NPY_ITFLAG_EXLOOP)  {
+        /* make sure to update the user pointers (copy to buffer does). */
```

Comment:
Above as well
```suggestion
        /* make sure to update the user pointers (copy to buffer already does). */
```

Reply:
adopted as part of the rebase (in both places).